### DISCUSSION TITLE
Allow verification of multiple domain names

### DISCUSSION THREAD

**Discussion by balint-front** [WalletConnect/walletconnect-monorepo#4362] [Ideas]
Hi there!
Are there any plans to support verifications of multiple domain names? We are building a dApp, that is hosted on multiple URLs, and our users are getting warnings in the wallets, when connecting to our site. We have one of our domain names verified using the DNS TXT record, but were wondering if multiple URLs will be allowed at some point? 

We have considered creating multiple Project IDs, but it's better to keep all of our traffic under one account, for statistics and managing our applications metadata. 
![Screenshot 2024-03-25 at 15 50 55](https://github.com/WalletConnect/walletconnect-monorepo/assets/81983602/a9fb8ad5-20e3-4368-a80d-c967b0c51c87)

**Comment by overplex-eth** [comment]
The existing documentation for WalletConnect does not specifically address support for confirming numerous domains under one account. So i think that is not possible

**Comment by Cali93** [comment]
Thanks for your feedback @balint-front.

It's not possible atm but we'll look into adding this in our roadmap.

**Comment by Cali93** [comment]
Hello @balint-front, support for verifying multiple domains has been added.
